CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8)
+
+add_subdirectory(src/)
```

Comment:
In general I always see add_subdirectory arguments without the path separator / . I think for consistency is good to omit it.